mock/cpp_mock_scenarios/src/merge/merge_left.cpp

```diff
@@ -30,9 +29,7 @@ class MergeLeftScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit MergeLeftScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "merge_left", ament_index_cpp::get_package_share_directory("kashiwanoha_map") + "/map",
```

Comment:
.osm file should be exactly fixed in scenario.
So, please specify it in constructor.